modules/channel-web/src/views/lite/components/messages/renderer/FileInput.tsx

```diff
@@ -38,7 +39,13 @@ export class FileInput extends React.Component<Renderer.FileInput> {
           onChange={() => {}}
           placeholder={this.props.placeholder}
           disabled={this.props.disabled}
-          style={{ position: 'relative', zIndex: -1 }}
+          style={{
+            position: 'absolute',
+            zIndex: -1,
+            width: '100%',
+            top: 0,
+            left: 0
+          }}
```

Comment:
can this be moved in a scss file, or in the .css file?

Reply:
It was like this but yes indeed we can